redi-connect-front/src/components/organisms/RediHeroProgram.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import classnames from 'classnames'
+import { Section, Container, Element, Heading, Content, Columns } from 'react-bulma-components'
+import { useTranslation } from 'react-i18next'
+import SVGImage from '../atoms/SVGImage'
+
+import './RediHeroProgram.scss'
+
+interface Props {
+  type: 'mentor' | 'mentee'
+}
+
+const programIcons = [
```

Comment:
What about using the `returnObjects` parameter from the i18n? You could move the icon names also to the translation file and make the code much more simple. This way you could completely decouple the content from the component.

https://www.i18next.com/translation-function/objects-and-arrays